cli/go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/op/go-logging v0.0.0-20160315200505-970db520ece7
 	github.com/spf13/cobra v1.3.0
 	github.com/spf13/pflag v1.0.5
-	github.com/up9inc/basenine/server/lib v0.0.0-20220302073458-c32e0adf1500
+	github.com/up9inc/basenine/server/lib v0.0.0-20220302182733-74dc40dc2ef0
```

Comment:
Can you remind me why the CLI imports basenine server module?

Reply:
@IgorGov It's introduced by https://github.com/up9inc/mizu/pull/843 specifically after this comment https://github.com/up9inc/mizu/pull/843#pullrequestreview-896123299

It's added as a dependency to do this check:

https://github.com/up9inc/mizu/blob/5f8a5a3a2947718156c1430f373d49a872932b9c/cli/config/configStructs/tapConfig.go#L82-L85